README.md

```diff
@@ -77,7 +82,7 @@ by the [PSR-3 logger interface standard](https://www.php-fig.org/psr/psr-3/#5-ps
 $logger = new LoggerClass();
 
 // Instantiate the client factory.
-$factory = new EPoetryClientFactory('http://europa.eu/epoetry.wsdl', $adapter);
+$factory = new RequestClientFactory('http://europa.eu/epoetry.wsdl', $adapter);
```

Comment:
Same as above, `RequestClientFactory` does not expect WSDL URI.

Reply:
True but out of scope. It also requires to change the command in `\OpenEuropa\EPoetry\Console\Command\CreateRequestsCommand` and it may require changes in other places. It will be better addressed in another PR.